.github/workflows/deploy-to-github-pages.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy:
+    name: Deploy to GitHub Pages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.15.0
+          cache: npm
+
+      - name: Install dependencies
+        run: npm ci
+      - name: Build website
+        working-directory: ./website
+        run: |
+          npm i
+          npm run build
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # Build output to publish to the `gh-pages` branch:
+          publish_dir: ./website/build
+          user_name: github-actions[bot]
+          user_email: 41898282+github-actions[bot]@users.noreply.github.com
```

Comment:
@yhy-1 who’s anonymized email is this?

Reply:
This was part of Docusaru's documentation.   https://docusaurus.io/docs/deployment#triggering-deployment-with-github-actions. It looks like https://github.com/actions/checkout/discussions/479#discussioncomment-625461 it shows GitHub action bot for the commit. Not sure the number is needed but it was in the documentation.

Reply:
K that works for me.